### ISSUE TITLE
snpEff database download locations do not exist for snpEff v4.4

### ISSUE THREAD

**Issue by bnat1** [pcingola/SnpEff#219]
Using snpEff 4.4 (latest, master), when I run: java -jar snpEff.jar download <any existing database name>
I get errors.
For example, for hg38:
```
$ java -jar snpEff-4.4.jar download hg38
00:00:00	ERROR while connecting to http://downloads.sourceforge.net/project/snpeff/databases/v4_4/snpEff_v4_4_hg38.zip
java.lang.RuntimeException: java.lang.RuntimeException: File not found on the server. Make sure the database name is correct.
	at org.snpeff.util.Download.download(Download.java:178)
	at org.snpeff.snpEffect.commandLine.SnpEffCmdDownload.downloadAndInstall(SnpEffCmdDownload.java:32)
	at org.snpeff.snpEffect.commandLine.SnpEffCmdDownload.runDownloadGenome(SnpEffCmdDownload.java:86)
	at org.snpeff.snpEffect.commandLine.SnpEffCmdDownload.run(SnpEffCmdDownload.java:72)
	at org.snpeff.SnpEff.run(SnpEff.java:1196)
	at org.snpeff.SnpEff.main(SnpEff.java:167)
Caused by: java.lang.RuntimeException: File not found on the server. Make sure the database name is correct.
	at org.snpeff.util.Download.download(Download.java:127)
	... 5 more
```
Sure enough, when I navigate to the url that was tried, nothing is there. Source forge redirects to
https://sourceforge.net/projects/snpeff/files/
And when I try navigating to the databases, there are no 4.4 files, only 4.3

**Comment by Lafond-LapalmeJ** [comment]
I have the same issue. Does anybody have a fix for this ? Or we just have to use the old 4.3 files?

**Comment by pcingola** [comment]
Closing old issues. Should be fixed in version 5.0
